cockroachcloud/pci-dss.md

```diff
@@ -0,0 +1,129 @@
+---
+title: Payment Card Industry Data Security Standard (PCI DSS) Compliance in CockroachDB Dedicated
+summary: Learn about compliance with the Payment Card Industry Data Security Standard (PCI DSS) for CockroachDB Dedicated clusters.
+toc: true
+docs_area: manage.security
+---
+
+The Payment Card Industry Data Security Standard (PCI DSS) is a minimum set of requirements for the safe handling of sensitive data associated with credit and debit cards. In the PCI DSS standard, this data is referred to as "cardholder data". When implemented correctly, PCI DSS helps to protect cardholder data from fraud, exfiltration, and theft. PCI DSS is mandated by credit card issuers but administered by the [Payment Card Industry Security Standards Council](https://www.pcisecuritystandards.org/).
+
+Many organizations who do not store cardholder data still rely upon compliance with PCI DSS to help protect other sensitive or confidential data or metadata.
+
+Responsibility for compliance with PCI DSS is shared among multiple parties, including card issuers, banks, software-as-a-service (SaaS) providers, and retail merchants. Compliance measures are implemented as a series of business practices, security controls, and technological solutions. An organization's compliance with PCI DSS is certified by a PCI Qualified Security Assessor (QSA).
+
+CockroachDB Dedicated has been certified by a QSA as a PCI DSS [Level 1 Service Provider](https://www.pcidssguide.com/what-are-pci-service-provider-compliance-levels/). This certification extends the existing [SOC 2 Type 2 certification](https://www.cockroachlabs.com/blog/soc-2-compliance-2/) of {{ site.data.products.dedicated }}, which provides a baseline level of security controls to safeguard customer data.
```

Comment:
Use the liquid variable for CockroachDB Dedicated here.

Also, I wonder if this doc should specify at the top that PCI DSS certification applies specifically to Dedicated. It currently reads more like a reference for PCI DSS in general rather than for how it affects Dedicated clusters.

Reply:
Alternatively, perhaps all the paragraphs preceding this one can be moved under the "Overview of PCI DSS" heading below?